pkg/server/singleprocess/service_job.go

```diff
@@ -70,25 +70,27 @@ func (s *Service) queueJobMulti(
 	ctx context.Context,
 	req []*pb.QueueJobRequest,
 ) ([]*pb.QueueJobResponse, error) {
-	jobs := make([]*pb.Job, 0, len(req))
+	jobQueue := make([]*pb.Job, 0, len(req)*4)
```

Comment:
I presume *4 is just a guess to so there isn't a slice resize operation?

Reply:
Yep.